README.md

```diff
@@ -346,6 +350,10 @@ var options = {
 };
 ```
 
+#### Passwordless options
+
+- **passwordlessMethod {String}**: When using `Auth0LockPasswordless` with an email connection, you can use this option to pick between sending a [code](https://auth0.com/docs/connections/passwordless/spa-email-code) or a [magic link](https://auth0.com/docs/connections/passwordless/spa-email-link) to authenticate the user. Available values are `code` and `link`. Defaults to `code`. Sms passworldess connections will always use `code`.
```

Comment:
- "[magic link]" => "[link]" to match the value
- "Available values are `code` and `link`" => "Available values for email connections are `code` (default) and `link`"
- "Sms" => "SMS"
- "passworldess" typo